### PR TITLE
fix: Remove content from image directives in README.rst

### PR DESCRIPTION
**Pull request by bassmang**

### CODE REVIEW

**Review by jackgerrits — APPROVED** [review]

File: python/README.rst [hunk 1]
```diff
@@ -1,18 +1,12 @@
...
-    :target: https://travis-ci.org/VowpalWabbit/vowpal_wabbit
-.. image:: https://ci.appveyor.com/api/projects/status/6hqpd9e64h72gybr/branch/master?svg=true
-    :alt: Windows Build Status
-    :target: https://ci.appveyor.com/project/VowpalWabbit/vowpal-wabbit
-.. image:: https://coveralls.io/repos/github/VowpalWabbit/vowpal_wabbit/badge.svg
-    :alt: Coverage
-    :target: https://coveralls.io/r/VowpalWabbit/vowpal_wabbit
+|pic1| |pic2| |pic3| |pic4|
+
+.. |pic1| image:: https://badge.fury.io/py/vowpalwabbit.svg
```
Comment (jackgerrits) [inline python/README.rst:6]: These are no longer clickable badges now though?

Comment (bassmang) [reply python/README.rst:6]: They should still be clickable

File: python/README.rst [hunk 2]
```diff
@@ -1,18 +1,12 @@
...
+.. |pic2| image:: https://travis-ci.org/VowpalWabbit/vowpal_wabbit.svg?
+.. |pic3| image:: https://ci.appveyor.com/api/projects/status/6hqpd9e64h72gybr/branch/master?svg=true
+.. |pic4| image:: https://coveralls.io/repos/github/VowpalWabbit/vowpal_wabbit/badge.svg
```
Comment (jackgerrits) [inline python/README.rst:9]: These no longer exist

File: python/README.rst (hunk 1 above)
Comment (jackgerrits) [reply python/README.rst:6]: Wont clicking it take you to https://badge.fury.io/py/vowpalwabbit.svg ?

Comment (bassmang) [reply python/README.rst]: Readded the link

File: python/README.rst [hunk 3]
```diff
@@ -1,7 +1,7 @@
 Vowpal Wabbit Python Wrapper
 ============================
 
-.. image:: https://badge.fury.io/py/vowpalwabbit.svg
+.. figure:: https://badge.fury.io/py/vowpalwabbit.svg
```
Comment (jackgerrits) [inline python/README.rst:4]: Could try this? https://stackoverflow.com/a/14387603/2214524

Comment (bassmang) [reply python/README.rst]: Yes there was an issue using a figure without a caption. I'll try this solution, and if it doesnt work I'll go back to figure and use an empty comment for the caption

**Pull request merged**